runtimes/aspnetcore-1.0/versions/1.0.9/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/google_appengine/debian8
+
+# Install .NET Core dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        libc6 \
```

Comment:
Are these taken from install instructions for .NET Core?

Reply:
Yep.